small-talk/db/registerTest.js

```diff
@@ -0,0 +1,28 @@
+"use server"
+const uri = "mongodb+srv://smt_root:pokemonwithguns@smalltalkcluster0.jo4jne6.mongodb.net/?retryWrites=true&w=majority"
+import Mongoboi from "./mongo"
+export default async function handleRegister(username, password, firstname, lastname, dob){
+  const mongoboi = new Mongoboi(uri, "Users");
+  await mongoboi.connect();
```

Comment:
Function works to create an entry in the databse for a new user. 

I would advise storing dob in unix time since it is more space efficient than storing a string.